connectors/src/connectors/github/temporal/activities.ts

```diff
@@ -846,6 +854,19 @@ async function garbageCollectCodeSync(
         },
       },
     });
+
+    // Also delete data source folders
+    const fq = new PQueue({ concurrency: 8 });
```

Comment:
here you'd use concurrentExecutor

Reply:
As discussed IRL, we stick to PQueue because it's used in this file, but we agree on using ConcurrentExecutor when not bound to any consistency constraint